bin/action_subscriber

```diff
@@ -11,6 +11,7 @@ module ActionSubscriber
     class_option :mode
     class_option :host
     class_option :hosts
+    class_option :prefetch, :type => :numeric, :desc => "how many messages to hold in the local queue in subscribe mode"
```

Comment:
I added a CLI option to set the prefetch just because it was convenient to my local testing, I can remove this if people think it should be in a different PR